otx/algorithms/anomaly/configs/base/configuration.py

```diff
@@ -48,6 +49,16 @@ class LearningParameters(ParameterGroup):
         header = string_attribute("Learning Parameters")
         description = header
 
+        # Editable is set to false as WideResNet50 is very large for
+        # onnx's protobuf (2gb) limit. This ends up crashing the export.
+        backbone = selectable(
```

Comment:
if the configurable parameters are changed, should the `configuration.yaml` files be updated as well?

Reply:
I think I was looking at a different PR for the diff. I have removed backbone from configuration.py and added the changed for nncf in configuration.py. But apart from that I don't think there are other changes between external and otx for DRAEM.